dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/metrics/BatchMetricsDispatcher.kt

```diff
@@ -43,7 +42,7 @@ internal class BatchMetricsDispatcher(
             internalLogger.logMetric(
                 messageBuilder = { BATCH_DELETED_MESSAGE },
                 additionalProperties = it,
-                samplingRate = MethodCallSamplingRate.LOW.rate
```

Comment:
this will increase the volume by 15x times, is it expected?

Reply:
Yep, this is to align with iOS metrics

Reply:
By looking on the volume for the "Batch Closed" telemetry for the latest SDK releases (2.17.0 for Android, 2.22.0 for iOS) the difference is like 8x, but only 3x for the 2.16.0 vs 2.21.0, so probably increasing this by 15x will create big disparity (and maybe this value should be lowered on iOS?), but if the goal is to have the same effective sampling rate between the 2 platforms, I'm fine with this change.